tutorial-701/acquia-pipelines.yaml

```diff
@@ -0,0 +1,32 @@
+version: 1.0.0
+
+variables:
+  global:
+    # Paste your n3_key from ~/.acquia/pipelines/credentials on the next line.
+    N3_KEY: 'your key'
+    # Run this command to securely add your Cloud API secret key:
+    #   awk '/n3_secret/ { printf("%s", $2) }' ~/.acquia/pipelines/credentials | pipelines encrypt - --add variables.global.N3_SECRET
+    PIPELINES_DEPLOY_URL: https://github.com/acquia/pipelines-examples/releases/download/deploy.1/pipelines-deploy
+    PIPELINES_DEPLOY_TOOL: $HOME/pipelines-deploy
+
+events:
+  build:
+    steps:
+      # If you have build instructions, put them here. They are not required.
+      - build_site:
+          script:
+            - echo "Put your build instructions, if any, here."
+
+      # Deploy the build artifact to a Cloud on-demand environment.
+      - deploy:
+          script:
+            - curl -o $PIPELINES_DEPLOY_TOOL "$PIPELINES_DEPLOY_URL" && chmod 755 $PIPELINES_DEPLOY_TOOL
+            - $PIPELINES_DEPLOY_TOOL
```

Comment:
Is there any documentation for pipelines-deploy? I might, generally as a developer, want to know what this binary does and maybe see some docs for it before I go about running it. And I'm not seeing it in the readme either.